modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -281,6 +281,11 @@ const register = (editor: Editor): void => {
   registerOption('sidebar_show', {
     processor: 'string'
   });
+
+  registerOption('help_accessibility', {
```

Comment:
If I understand this correctly, the reason for not registering this option in `help` plugin is because it can't be accessed/used in theme. If so, it is worth having a comment explaining why

Reply:
Yep, added a comment explaining why.